modules/aggregation-layers/src/contour-layer/marching-squares-codes.js

```diff
@@ -149,6 +157,37 @@ export const ISOLINES_CODE_OFFSET_MAP = {
   15: []
 };
 
+export const LINEAR_INTERPOLATION_CODE_OFFSET_MAP = {
```

Comment:
Is this map same as `ISOLINES_CODE_OFFSET_MAP` ? if same remove, if not can you explain how it differs and update the comments to understand what it is mapping.

Reply:
We have four weights for a single cell in below format 
```
weights = {
top: 10,
right: 12,
current: 8,
topRight: 14
}
```
So this map will return the keys array `(top, topRight, current, right)` bypassing cell code value. using these keys, I am extracting the weights and calculate the Linear interpolation for that code.

I also change the description of this map.